src/admin/mod.rs

```diff
@@ -203,6 +206,7 @@ impl Server {
                                 "/metrics" => {
                                     Ok::<_, hyper::Error>(handle_metrics(registry, req).await)
                                 }
+                                "/loglevel" => Ok::<_, hyper::Error>(handle_loglevel(req).await),
```

Comment:
```suggestion
                                "/logging" => Ok::<_, hyper::Error>(handle_loglevel(req).await),
```

lets mirror Envoy's API: https://www.envoyproxy.io/docs/envoy/latest/operations/admin#post--logging

Reply:
done